client/task-list/tasks/payments/PaymentGatewaySuggestions/components/Setup/Setup.js

```diff
@@ -103,6 +104,9 @@ export const Setup = ( {
 						<Plugins
 							onComplete={ ( installedPlugins, response ) => {
 								createNoticesFromResponse( response );
+								invalidateResolutionForStoreSelector(
```

Comment:
I guess this is necessary since we no longer fetch the individual gateway after plugin install, is that right?

Reply:
Yup, need to ensure that this request is rerun to include the enriched props.